packages/theme-addresses/__tests__/theme-addresses.test.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * @jest-environment jsdom
+ */
+import {
+  CountryProvinceSelector
```

Comment:
Did Prettier make this newline? Seems like overkill.

Reply:
Nah .. I think it was just copy paste laziness